fees/llamalend.ts

```diff
@@ -0,0 +1,61 @@
+import { Adapter } from "../adapters/types";
+import { CHAIN } from "../helpers/chains";
+import { request, gql } from "graphql-request";
+import type { ChainEndpoints } from "../adapters/types"
+import { Chain } from '@defillama/sdk/build/general';
+import { getTimestampAtStartOfDayUTC } from "../utils/date";
+import BigNumber from "bignumber.js";
+import { getBlock } from "../helpers/getBlock";
+import { getPrices } from "../utils/prices";
+
+const endpoints = {
+  [CHAIN.ETHEREUM]: "https://api.thegraph.com/subgraphs/name/0xngmi/llamalend",
+}
+const ONE_DAY_IN_SECONDS = 60 * 60 * 24
+
+interface IGraph {
+  interest: string;
+  borrowed: string;
+}
+const graphs = (graphUrls: ChainEndpoints) => {
+  return (chain: Chain) => {
+    return async (timestamp: number) => {
+      const todaysTimestamp = getTimestampAtStartOfDayUTC(timestamp)
```

Comment:
why use getTimestampAtStartOfDayUTC()? id either use the timestamp directly or timestamp -24h